playbooks/roles/load-balancer-v2/files/prod/X29uZGVtYW5kLmNsb3VkZXJhLmNvbQ==.pem

```diff
@@ -0,0 +1,276 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
@SSPJ Just a quick note: I thought that these certificates were going to be stored on `ansible-common-secrets`.

Reply:
@giovannicimolin No, the whole reason for introducing Ansible Vault was so that I could deploy these just like ordinary files.

Reply:
:+1: